### PR TITLE
Add contribution guideline.

### PR DESCRIPTION
**Pull request by objmagic**
See https://github.com/blog/1184-contributing-guidelines

### CODE REVIEW

**Comment by objmagic** [comment]
Preview effect:

image

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1 @@
+website/content/docs/contributors/community.md
```
Comment (billonahill) [inline CONTRIBUTING.md]: Could we link to the site at httpp://heronstreaming.io/docs/contributors/community/ or  http://twitter.github.io/heron/docs/contributors/community/ or would that not work?

Comment (objmagic) [inline CONTRIBUTING.md]: I created a symlink called `CONTRIBUTING.md` at root directory pointing to [website/content/docs/contributors/community.md](https://github.com/twitter/heron/blob/master/website/content/docs/contributors/community.md). I don't know why GitHub diff displays something like that.

Comment (objmagic) [inline CONTRIBUTING.md]: we cannot link to site. We have to put a `CONTRIBUTING.md` (symlink fine) under root.

**Comment by billonahill** [comment]
👍

**Pull request merged**